source/SquadLeader/res/xml/chem_light_callout_style.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+<!--calloutViewStyle
```

Comment:
Let's keep it. It will make it easier if we decide we want to style it differently in the future.